gtsfm/data_association/data_assoc.py

```diff
@@ -33,6 +33,8 @@
 
 logger = logger_utils.get_logger()
 
+MAX_DELAYED_TRIANGULATION_CALLS = 1e7
```

Comment:
nit: could we add a brief inline comment explaining this variable perhaps:
```python
# Heuristically set to limit the number of nodes in the Dask graph.
MAX_DELAYED_TRIANGULATION_CALLS = 1e7
```

Reply:
Done